Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+class AppStoreConnectService {
+    private let provider: APIProvider
+
+    init(configuration: APIConfiguration) {
+        provider = APIProvider(configuration: configuration)
+    }
+
+    func listUsers(with options: ListUsersOptions) -> AnyPublisher<[User], Error> {
```

Comment:
If this file getting bigger, we can probably separate these request functions into different files in the future